addon/templates/components/nypr-o-gallery-overlay/slide.hbs

```diff
@@ -61,6 +61,12 @@
               @ariaLabel="Share on Facebook"
               @utm={{@share.facebook.utm}}
             />
+            <tools.share
+              @service='spotify'
+              @url={{this.shareURL}}
+              @ariaLabel="Share with Spotify"
+              @utm={{@share.spotify.utm}}
+            />
```

Comment:
this isn't quite right; doing this is going to add a spotify button to all gallery slides